doofinder.php

```diff
@@ -2754,4 +2754,11 @@ protected function getIsoCodeById($id)
             SELECT `iso_code` FROM ' . _DB_PREFIX_ . 'currency WHERE `id_currency` = ' . (int) $id
         );
     }
+
+    protected function getLanguageCode($code){
+        // $code is in the form of 'xx-YY' where xx is the language code
+        // and 'YY' a country code identifying a variant of the language.
+        $lang_country = explode('-', $code);
+        return $lang_country[0];
```

Comment:
Is the format always xx-xx?
Would it be safer get the first two chars?

Reply:
**xx-yy** or **xx** only. Both cases are contemplated with the function
Prestashop uses it in one of its classes: https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Language.php#L959

Reply:
ok